cmd/loki/Dockerfile.debug

```diff
@@ -1,8 +1,9 @@
 # Directories in this file are referenced from the root of the project not this folder
 # This file is intended to be called from the root like so:
 # docker build -t grafana/loki -f cmd/loki/Dockerfile.debug .
+ARG BUILD_IMAGE
```

Comment:
I'm really unfamiliar with ARG but could you somehow have a default value for it? So that we could keep as default the value `grafana/loki-build-image`

Reply:
I tried to get a default value working, but it proved tricky.
I've just hardcoded the image version now as it is everywhere else too.